src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -88,6 +88,8 @@
     PARSE_TEX(Localization.lang("Search for Citations in LaTeX Files"), IconTheme.JabRefIcons.LATEX_CITATIONS),
     NEW_SUB_LIBRARY_FROM_AUX(Localization.lang("New sublibrary based on AUX file") + "...", Localization.lang("New BibTeX sublibrary") + Localization.lang("This feature generates a new library based on which entries are needed in an existing LaTeX document."), IconTheme.JabRefIcons.NEW),
     WRITE_XMP(Localization.lang("Write XMP-metadata to PDFs"), Localization.lang("Will write XMP-metadata to the PDFs linked from selected entries."), KeyBinding.WRITE_XMP),
+    XMP_FILTER_ADD(Localization.lang("Add"), Localization.lang("Add field to filter list"), IconTheme.JabRefIcons.ADD_NOBOX),
```

Comment:
This enum should only contain major actions that the user can perform (e.g. display of dialogs) but not each small button action. See below for a suggestion of how to resolve this.